schemas/oic.oic-link-schema.json

```diff
@@ -118,14 +112,30 @@
         "type": {
           "type": "array",
           "description": "A hint at the representation of the resource referenced by the target URI. This represents the media types that are used for both accepting and emitting",
-          "items" : [
-            {
+          "items" : {
             "type": "string",
             "maxLength": 64
-            }
-          ],
+            },
           "minItems": 1,
           "default": "application/cbor"
+        },
+        "eps": {
+          "type": "array",
+          "description": "the Endpoint information of the target Resource",
+          "items": {
+            "type": "object",
+            "properties": {
+              "ep": {
+                "type": "string",
+                "format": "uri",
+                "description": "URI with Transport Protocol Suites + Endpoint Locator as specified in 10.2.1"
+              },
+              "pri": {
```

Comment:
The spec states that pri is a positive integer; so shouldn't this be:
"type": "integer"
"minimum": 1